package.json

```diff
@@ -60,27 +60,21 @@
     "react-dom": "^15.0.1"
   },
   "devDependencies": {
-    "babel": "^6.5.2",
-    "babel-cli": "^6.7.5",
-    "babel-core": "^6.18.2",
-    "babel-eslint": "^7.1.0",
-    "babel-jest": "^17.0.2",
-    "babel-loader": "^6.2.4",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.10.3",
-    "babel-plugin-transform-es2015-modules-commonjs-simple": "^6.7.4",
-    "babel-plugin-transform-react-constant-elements": "^6.9.1",
-    "babel-plugin-transform-react-inline-elements": "^6.8.0",
-    "babel-plugin-transform-runtime": "^6.15.0",
-    "babel-preset-philpl": "^0.4.0",
-    "eslint": "^3.9.1",
-    "eslint-config-airbnb-base": "^9.0.0",
-    "eslint-plugin-import": "^2.1.0",
-    "eslint-plugin-react": "^6.5.0",
+    "@types/jest": "^18.1.1",
+    "@types/mapbox-gl": "^0.29.0",
+    "@types/node": "^7.0.5",
```

Comment:
If you remove every `require` you don't need this anymore (+ it's a bit overkill to add node types definition just for require)